src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -16,48 +19,135 @@ export interface BreadcrumbItem {
 export interface BreadcrumbProps extends Pick<LinkProps, 'as'> {
   /** A list of `BreadcrumbsItem` objects ( `{href,children}` ) that will construct the Breadcrumb */
   items: BreadcrumbItem[];
+  /** Truncates component if breadcrumb list is too long.
+   * At least 4 breadcrumbs must exist for truncation to occur.
+   */
+  truncate?: boolean;
 }
 
+interface IBreadcrumbLink {
+  crumb: BreadcrumbItem;
+  isLastBreadcrumb?: boolean;
+  showChevron?: boolean;
+}
+
+type ITruncatedBreadcrumbs = Omit<BreadcrumbProps, 'truncate'>;
+
+const BreadcrumbChevron: React.ElementType = () => {
+  return <Icon type="chevron-right" mx={2} color="blue-300" size="medium" role="presentation" />;
+};
+
+const BreadcrumbLink: React.ElementType = ({
+  crumb,
+  isLastBreadcrumb = false,
+  showChevron = true,
+  ...rest
+}: IBreadcrumbLink) => {
+  return (
+    <Flex
+      key={crumb.href}
+      as="li"
```

Comment:
Thank you for pointing this out because this element should always be the child of a list. I've removed `<Card key={item.href} minWidth={180} pb={5}>` to make this true.